Snippets/Rabbit/Rabbit_4/Usage.cs

```diff
@@ -140,4 +141,14 @@ void PrefetchCount(EndpointConfiguration endpointConfiguration)
         #endregion
     }
 
+    void SetClientCertificates(EndpointConfiguration endpointConfiguration)
+    {
+        #region rabbitmq-config-client-certificates
+
+        var transport = endpointConfiguration.UseTransport<RabbitMQTransport>();
+        transport.SetClientCertificates(new X509CertificateCollection());
```

Comment:
@adamralph What do you think would be better? Creating a new `X509Certificate2`, adding it to a new collection, and then passing in the collection?

I'm not sure if that would be better or not, because if you're wanting to use this API, you're likely already getting your certificate instances from somewhere else, not creating them just to pass them in.